### PR TITLE
chore: Update dropdown placement and alignment

### PR DESCRIPTION
**Pull request by ogzcode**
Dropdown doc problem fixed: #312

Code labels have been rearranged in accordance with the sample components

Please check and approve this PR. I will be waiting for your feedback.

@Sqrcz @cogor @zoltanszogyenyi @robert1508

## Summary by CodeRabbit

- **Documentation**
	- Updated the dropdown component documentation to reflect changes in the text and placement properties for improved clarity on visual representation.

### CODE REVIEW

**Comment by Sqrcz** [comment]
Hey, 

I'll be back from longer leave tomorrow.

I'll go through the changes and let you know. Thanks 🙏

File: docs/components/dropdown.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The dropdown component can be used to show a list of menu items when clicking on
 <fwb-dropdown-example-placement />
 ```vue
 <template>
-  <fwb-dropdown text="Bottom">
+  <fwb-dropdown text="Top">
```
Comment (Sqrcz) [inline docs/components/dropdown.md:25]: It also requires `placement="top"` to show on top... as in the example file here: `docs/components/dropdown/examples/FwbDropdownExampleAlignment.vue`

File: docs/components/dropdown.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ The dropdown component can be used to show a list of menu items when clicking on
       </p>
     </div>
   </fwb-dropdown>
-  <fwb-dropdown placement="right" text="Right">
+  <fwb-dropdown placement="bottom" text="Bottom">
```
Comment (Sqrcz) [inline docs/components/dropdown.md:57]: we can remove `placement="botttom"` as it is the default position

File: docs/components/dropdown.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ The property controls how the dropdown is aligned with the trigger
 <fwb-dropdown-example-alignment />
 ```vue
 <template>
-  <fwb-dropdown text="Bottom" align-to-end>
+  <fwb-dropdown text="Top" align-to-end>
```
Comment (Sqrcz) [inline docs/components/dropdown.md:103]: same as above.

File: docs/components/dropdown.md [hunk 4]
```diff
@@ -132,7 +132,7 @@ The property controls how the dropdown is aligned with the trigger
       </p>
     </div>
   </fwb-dropdown>
-  <fwb-dropdown placement="right" text="Right" align-to-end>
+  <fwb-dropdown placement="bottom" text="Bottom" align-to-end>
```
Comment (Sqrcz) [inline docs/components/dropdown.md:135]: same as above.

**Review by Sqrcz — CHANGES_REQUESTED** [review]
Hey,

Nice catch... thank you.

Please amend those small little fixes above 🙏 and then we can merge it.

**Comment by ogzcode** [comment]
Hello, I have completed the conversations, can you check? @Sqrcz

**Review by Sqrcz — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**